app.js

```diff
@@ -1,9 +1,26 @@
 const config = require('./config/conf.json');
-//import config from './config/conf.json';
 const models = require('./models/models');
+const DirWatcher = require('./modules/dirwatcher').default;
+const Importer = require('./modules/importer').default;
+
+function getValue(flag) {
+    const index = process.argv.indexOf(flag);
+    return (index > -1) ? process.argv[index + 1] : null;
+}
+
+const path = __dirname + (getValue('-dir')|| ('/data/'));

```

Comment:
Please consider using any other name (maybe more specific) as this matches to the default `path` module name so this may be confusing